out/command.go

```diff
@@ -84,7 +85,10 @@ func (c *Command) Run(input concourse.OutRequest) (concourse.OutResponse, error)
 			varsFilepaths = append(varsFilepaths, varFilepath)
 		}
 
-		_, err = c.flyCommand.SetPipeline(p.Name, configFilepath, varsFilepaths, p.Vars)
+		var setOutput []byte
+		setOutput, err = c.flyCommand.SetPipeline(p.Name, configFilepath, varsFilepaths, p.Vars)
+		c.logger.Debugf("Set for pipeline: %s.\n Output: \n %s", p.Name, string(setOutput))
+		fmt.Fprintf(os.Stderr, "Set for pipeline: %s.\n Output: \n %s", p.Name, string(setOutput))
```

Comment:
The output seems a little weird, how about something like:

```suggestion
		fmt.Fprintf(os.Stderr, "pipeline '%s' set; output:\n\n%s\n", p.Name, string(setOutput))
```

Alternatively we could just not capture the output and let it go directly to stderr, but that would probably require changes to the fly command object.

Reply:
Thank you for your comment! I've just fixed it!